rtgui/options.cc

```diff
@@ -343,15 +343,15 @@ void Options::setDefaults()
     dirBrowserSortType = Gtk::SORT_ASCENDING;
     preferencesWidth = 800;
     preferencesHeight = 600;
-    toolPanelWidth = 400;
+    toolPanelWidth = 465;
     browserToolPanelWidth = 465;
     browserToolPanelHeight = 600;
     browserToolPanelOpened = true;;
     browserDirPanelOpened = true;
     editorFilmStripOpened = true;
     historyPanelWidth = 330;
     fontFamily = "default";
-    fontSize = 10;
+    fontSize = 8;
```

Comment:
Why, based on what tests? If this is solely to accommodate Local Adjustment's horizontal space problem, :-1: